.github/workflows/python_compile_requirements.yml

```diff
@@ -1,7 +1,7 @@
 name: Compile python requirements
 on:
   schedule: 
-  - cron: 0 5 * * SUN
+  - cron: 0 17 * * SAT
```

Comment:
These are UTC if that wasn't clear! So moved up 12 hr from Sunday 0:00 to Sat 12:00

Reply:
(not for this PR) I wonder if we can put time zone in these